renku/cli/__init__.py

```diff
@@ -200,6 +202,16 @@ def cli(ctx, path, renku_home, use_external_storage):
         use_external_storage=use_external_storage,
     )
 
+    if (
+        path != os.getcwd() and
+        ctx.invoked_subcommand not in WARNING_UNPROTECTED_COMMANDS
+    ):
+        click.secho(
```

Comment:
One small improvement would be to show this message after the command output and not before. For example I have a repo with ~30 datasets and when I do `renku datasets` it's quite easy to miss this warning message.

Reply:
TBH, I don't know a clean way of showing the message at the end using Click (considering pagers as well). This is the case when there is an exception and the bug report prompt is show (where this warning is probably needed most).